make.js

```diff
@@ -1459,7 +1459,14 @@ target.lint = function() {
   var jshintPath = path.normalize('./node_modules/.bin/jshint');
   if (!test('-f', jshintPath)) {
     echo('jshint is not installed -- installing...');
-    exec('npm install jshint@2.4.x'); // TODO read version from package.json
+    // Read the jshint version to be installed from package.json.
+    try {
+      var rawConfiguration = fs.readFileSync('package.json', 'utf8');
+      var configuration = JSON.parse(rawConfiguration);
+      exec('npm install jshint@' + configuration.devDependencies.jshint);
+    } catch(e) {
+      echo('package.json does not exist -- aborting...');
```

Comment:
Nit: You might want to `return` or `throw` after this line, so that the code actually aborts, otherwise the print statement is slightly wrong :-)

Super-nit: How about adding a space between `catch` and `(e) {` above? :-P